colab/edge_detection.ipynb

```diff
@@ -403,36 +404,44 @@
       "source": [
         "#@title Compile and prepare to test the edge detection module\n",
         "\n",
-        "flatbuffer_blob = compile_str(compiler_module, target_backends=[\"vmvx\"], input_type=\"mhlo\")\n",
+        "flatbuffer_blob = compile_str(compiler_module, target_backends=[\"dylib\"], input_type=\"mhlo\")\n",
         "vm_module = ireert.VmModule.from_flatbuffer(flatbuffer_blob)\n",
         "\n",
         "# Register the module with a runtime context.\n",
         "config = ireert.Config(backend.driver)\n",
         "ctx = ireert.SystemContext(config=config)\n",
         "ctx.add_vm_module(vm_module)"
       ],
-      "execution_count": 11,
+      "execution_count": null,
       "outputs": []
     },
     {
       "cell_type": "code",
       "metadata": {
         "colab": {
           "base_uri": "https://localhost:8080/",
-          "height": 247
+          "height": 264
         },
         "id": "OUUXxol7wl-f",
-        "outputId": "463c4b3b-40f5-4569-ce4a-53accbd88040"
+        "outputId": "26419038-b72b-494e-ec1c-abfaa2aac1df"
       },
       "source": [
         "edge_detect_sobel_operator_f = ctx.modules.module[\"edge_detect_sobel_operator\"]\n",
         "\n",
         "low_level_iree_edges = edge_detect_sobel_operator_f(image)\n",
         "\n",
-        "show_images(image, low_level_iree_edges)"
+        "#print(low_level_iree_edges)\n",
```

Comment:
might want to leave out the `#print(low_level_iree_edges)` (or toggle the comment on it if it's useful)